src/plugins/mer/mersettings.cpp

```diff
@@ -123,6 +163,130 @@ QMap<QString, MerEmulatorDeviceModel> MerSettings::deviceModelsRead(const Utils:
     return result;
 }
 
+FileName MerSettings::deviceModelsFileName()
```

Comment:
It would be nice to see this defined immediately after globalDeviceModelsFileName